lib/board.rb

```diff
@@ -0,0 +1,39 @@
+class Board
+  def initialize
+    @board = [
+      %w[1 2 3],
+      %w[4 5 6],
+      %w[7 8 9]
+    ]
+  end
+
+  def display_board
+    i = 0
+    while i < 3
+      x = 0
+      while x < 3
```

Comment:
There is a very useful method called `times` that is preferred in ruby. You could try it here. Check this [resource](https://www.rubyguides.com/ruby-tutorial/loops/) about the best loop ruby methods.